libgalois/include/katana/PropertyGraph.h

```diff
@@ -29,12 +29,81 @@ struct KATANA_EXPORT GraphTopology {
   using edges_range = StandardRange<edge_iterator>;
   using iterator = node_iterator;
 
+private:
+  LargeArray<Edge> adj_indices_;
+  LargeArray<Node> dests_;
```

Comment:
nit: use consistent names w.r.t. arrow arrays below. Either add `_` or `array_` to those names.

Reply:
Also it would be better to use `adj_out_dests` for consistency

Reply:
I want to use _ ending names for all members, however, for compatibility with existing code, including Python, I've revered arrow arrays to their old names.